app/views/recipes/generate_shopping_list.html.erb

```diff
@@ -0,0 +1,33 @@
+<div class="container">
+  <h2 class="mt-4">Generate Shopping List</h2>
+  <div class="row mt-4">
+    <div class="col-md-6">
+      <p>Amount of food items to buy: <%= @required_foods.to_a.count %></p>
+      <p>Total Value of food needed: <%= number_to_currency(@total_value) %></p>
+    </div>
+  </div>
+
+  <h3 class="mt-4">Required Food Items</h3>
+  <table class="table table-bordered mt-2">
+    <thead>
+      <tr>
+        <th>Name</th>
+        <th>Quantity</th>
+        <th>Measurement Unit</th>
+        <th>Price</th>
+      </tr>
+    </thead>
+    <tbody>
+      <% @required_foods.each do |food| %>
+        <tr>
+          <td><%= food.name %></td>
+          <td><%= food.quantity %></td>
+          <td><%= food.measurement_unit %></td>
+          <td><%= number_to_currency(food.quantity.to_i * food.price.to_f) %></td>
+        </tr>
+      <% end %>
+    </tbody>
+  </table>
+
+  <%= link_to 'Back to Recipe', recipe_path(@recipe), class: 'btn btn-primary mt-4' %>
+</div>
```

Comment:
- [ ] [REQUIRED] The recipe and the inventory part of this table are missing. Should show the list of food that is missing for all recipes of the logged-in user. Kindly implement as stated by the project requirement.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="1327" alt="Screenshot 2023-09-16 at 22 59 51" src="https://github.com/techmoves/recipe-app/assets/58692590/071a6527-b25e-4223-9eaa-60bf9839f46a"> | <img width="1177" alt="Screenshot 2023-09-16 at 23 00 09" src="https://github.com/techmoves/recipe-app/assets/58692590/af2e5e50-1b44-4119-9466-a6d6cefa1d2d">